src/main/frontend/src/components/landingpage/FeatureItem.js

```diff
@@ -0,0 +1,14 @@
+import classes from './FeatureItem.module.css';
+import ItemImage from '../../assets/featurepic.png'
+
+function FeatureItem(){
+    return(
+        <div className={classes.featureitembox}>
+            <img src={ItemImage}/>
```

Comment:
Same as category, this component needs to receive Item as prop and just display it
```suggestion
            <img src={item.image}/>
            <h3 className={classes.cardtitle}>{ item.name }</h3>
            <p className={classes.carddescription}>Start from <span className={classes.cardprice}>${ item.price }</span></p>
```